pkgs/by-name/gk/gk-cli/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+}:
+
+let
+  version = "2.1.0";
+in
+stdenv.mkDerivation {
+  pname = "gk-cli";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/gitkraken/gk-cli/releases/download/v${version}/gk_${version}_Linux_x86_64.zip";
+    hash = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
+    stripRoot = false;
+  };
```

Here's my take on supporting multiple platforms, not that I've left out several of the platforms that are available upstream in the example below, yet I'd recommend adding them to this PR.
```suggestion
  src = let
    source = {
      x86_64-linux = {
        system = "Linux_x86_64";
        hash = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
      };
      aarch64-darwin= {
        system = "macOS_arm64";
        hash = "sha256-0zXnZcA0M1qz9c0cSQExri9CtqShPj3HTbkgS1Lsf5k=";
      };
      # TODO: Add entries for other supported gk-cli systems and their architecture
    }.${stdenv.system};
  in fetchzip {
    inherit (source) hash;
    url = "https://github.com/gitkraken/gk-cli/releases/download/v${finalAttrs.version}/gk_${finalAttrs.version}_${source.system}.zip";
    stripRoot = false;
  };
```